.yarnrc.yml

```
@@ -0,0 +1,12 @@
+enableTransparentWorkspaces: false
+
+nodeLinker: node-modules
```

```suggestion
nodeLinker: node-modules
enableGlobalCache: true
```
Since you're using the `node-modules` linker you can enable the global cache to speed up installs
https://yarnpkg.com/configuration/yarnrc#enableGlobalCache

---

Can I assume the global cache folder is `$HOME/.yarn/cache`? I have checked on https://yarnpkg.com/configuration/yarnrc but didn't find out. It would be great if the global cache folder can be obtained programmatically so we don't have to hardcode.

---

> Can I assume the global cache folder is $HOME/.yarn/cache?

Almost, it's located in `~/.yarn/berry/cache` on unix and `~/AppData/Local/Yarn/Berry/cache/` on Windows

> It would be great if the global cache folder can be obtained programmatically so we don't have to hardcode.

If you have the `enableGlobalCache: true` in your config then `yarn config get cacheFolder` points to it